DatabaseWorkshop.md

```diff
@@ -1,3 +1,22 @@
+# Building a "ToDo" Backend with Kitura
+
+<p align="center">
+<img src="https://www.ibm.com/cloud-computing/bluemix/sites/default/files/assets/page/catalog-swift.svg" width="120" alt="Kitura Bird">
+</p>
+
+<p align="center">
+<a href= "http://swift-at-ibm-slack.mybluemix.net/">
+    <img src="http://swift-at-ibm-slack.mybluemix.net/badge.svg"  alt="Slack">
+</a>
+</p>
+
+## Workshop Table of Contents:
```

Comment:
Can we indicate which stage of the workshop this is?  Perhaps put the current number in bold or something?